src/coreclr/inc/cordebuginfo.h

```diff
@@ -2,7 +2,7 @@
 // The .NET Foundation licenses this file to you under the MIT license.
 
 //
-// Keep in sync with https://github.com/dotnet/corert/blob/master/src/Native/ObjWriter/cordebuginfo.h
+// Keep in sync with https://github.com/dotnet/llvm-project/blob/objwriter/<branch>/llvm/tools/objwriter/cordebuginfo.h
```

Comment:
Its likely my lack of familiarity with the` llvm-project` repo but I found it a little challenging to find the `coredebuginfo.h` file there

Reply:
I have updated the comment that should make navigating to the right file easier.